Cargo.toml

```diff
@@ -33,6 +33,7 @@ tracing-tracy = ["tracing", "dep:tracing-tracy"]
 tokio-tracing = ["tokio/tracing"]
 stacktrace = ["rstack-self"]
 chaos-testing = []
+data-consistency-check = ["collection/data-consistency-check"]
```

Comment:
This is very much a nitpick, but I can imagine us potentially adding more of these in the future.

Shall we prefix them all with `debug-` to organize it a bit?

Reply:
This sounds sensible.
Do you want me to rename everything in this PR or later?